manifest.yml

```diff
@@ -0,0 +1,12 @@
+manifest_version: 1
+name: ImageTorchEncoder
+kind: encoder
+type: pod
```

Comment:
remove this I think

Reply:
https://github.com/jina-ai/executor-image-torch-encoder/pull/1/commits/aa73514a926a12274e914fd98e51e1605f46b7af